xapian-applications/omega/docs/postprocessing.rst

```diff
@@ -0,0 +1,42 @@
+=========================
+Log Data Postprocessing
+=========================
+
+Log data includes a) data logged from the search results page rendered by
+Omega query template and, b) clicks recorded when a document link is
+clicked on the results page.
+
+Further processing of this raw data is handled by the ``postprocess.py``
```

Comment:
Not sure about the name before the `.`, but I would omit the extension (both from the docs and also from the script itself).  The fact this is written in Python is entirely an implementation detail, and it might end up rewritten in C++ or a different scripting language in the future.  At that point, we have the choice of either dropping the `.py` and breaking any scripts, cronjobs, etc users have which call this program, and making all references to the script name in docs, blog posts, QA sites, etc out of date, or else we have a program called `postprocess.py` which isn't written in Python, which would be strange and confusing.

This is particularly true of scripts which get installed, though I've tried to avoid extensions on scripts in the source tree in general, except when those scripts are inherently specific to a language (as in the bindings) or when there's a hard to avoid reason for needing an extension.